src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.scss

```diff
@@ -12,3 +12,8 @@
     font-size: 14px;
     line-height: 20px;
 }
+
+// overwrite z-index
+.bp-AnnotationsTooltip-element.tooltip-element {
+    z-index: 160;
```

Comment:
It seems like this value is very specific to the context where Preview is being used. Can we move the override to that repo?